app/views/layouts/application.html.slim

```diff
@@ -10,14 +10,11 @@ html
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
 
   body
-    - if user_signed_in?
-      = link_to 'Log out', destroy_user_session_path, method: :delete
-    - else
-      = link_to 'Log in', new_user_session_path
+    = render 'shared/navbar'
 
-    == yield
-    
-    p.notice
-      = notice
-    p.alert
-      = alert
+    div.container
+      == yield
+      p.notice
```

Comment:
Just a cosmetic suggestion.
Intuitively feeling that it is better to show errors right after the navbar.
And what do you think about wrapping it into fancy css-class like `.alert` which bootstrap [provides](https://getbootstrap.com/docs/4.0/components/alerts/)?